public/components/Main/main.tsx

```diff
@@ -132,7 +132,14 @@ export function getQueryResultsForTable(queryResults: ResponseDetail<string>[]):
           case 'describe':
           case 'default':
             for (const [id, field] of schema.entries()) {
-              fields[id] = _.get(field, 'name');
+              let alias: any = null;
+              try {
+                alias = _.get(field, 'alias');
+              } catch (e) {
+                console.log("No alias for field " + field);
```

Comment:
Np. I suggest using either single quote or double quote for `js`, not both. One reason I am more for single quote is 
> A single quoted string can have double quotes within it without having to escape them

Reply:
Good point, thanks for raising it!